config-ui/src/pages/connections/webhook/form-modal.jsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import React, { useState, useEffect, useMemo } from "react";
+import { Dialog, Button, Toaster, Position, Intent } from "@blueprintjs/core";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+import { ReactComponent as Copy } from "@/images/icons/copy.svg";
+import * as S from "./styled";
+
+const CopyToaster = Toaster.create({
+  position: Position.TOP_RIGHT,
+});
+
+export const FormModal = ({
```

Comment:
Let's refactor this to `WebhookFormModal` and relocate file to `@/components/WebhookModalForm.jsx'

```suggestion
export const WebhookFormModal = ({
```

Reply:
I think the form modal is a component of this directory, not a common component.